File: code/MATLAB/Files/SocialNetworksMATLAB/genPeopleLocConnMat.m
function [ connMat ] = genPeopleLocConnMat( maxPersonId, locs, activities )
%GENPEOPLELOCCONNMAT function to generate people-to-location graph
%   Function returns adjacency matrix with people indices as row labels 
%   and location indices as column labels

connMat = zeros(maxPersonId, length(locs));
for i = 1:length(activities)
    connMat(activities(i, 2), index(activities(i, 7) , locs)) = 1;
end

end

File: products/Introduction_to_Computational_Science_Modules/02_System_Dynamics/R/R_Computational_Toolbox_Files/SocialNetworks_R/genPeopleLocConnMat.R
genPeopleLocConnMat<-function( maxPersonId, locs, activities ) {
#GENPEOPLELOCADJMAT function to generate people-to-location graph
#   Function returns adjacency matrix with people indices as row labels 
#   and location indices as column labels

connMat = matrix(0,maxPersonId,length(locs))
for (i in 1:nrow(activities)) {
    connMat[activities[i, 2], locationIndex(activities[i, 7], locs)] = 1
}
return (connMat)
}

